salt/modules/pkgin.py

```diff
@@ -190,7 +192,7 @@ def latest_version(*names, **kwargs):
                 s = _splitpkg(p[0])
                 if s:
                     if not s[0] in pkglist:
-                        if len(p) > 1 and p[1] == '<':
+                        if len(p) > 1 and p[1] == '<' or p[1] == '':
```

Comment:
``p[1]`` will be an empty string here if the package has not yet been installed.